lua/plenary/async/tests.lua

```diff
@@ -7,7 +7,7 @@ M.describe = function(s, async_func)
 end
 
 M.it = function(s, async_func)
-  it(s, util.will_block(async_func))
+  it(s, util.will_block(async_func, 20000)) -- TODO: use test timeout
```

Comment:
Not sure how we can access this. Maybe the test runner should set it as a `PLENARY_TEST_TIMEOUT` env var, which we can read here?

```suggestion
  it(s, util.will_block(async_func, tonumber(vim.env.PLENARY_TEST_TIMEOUT)))
```

Reply:
not a bad idea. Lets do it